Surface_sweep_2/test/Surface_sweep_2/cgal_test_base

```diff
@@ -88,14 +99,14 @@ clean_tests()
 {
     if [ "${TEST_WITH_CMAKE}" != "FALSE" ]; then
     #
-    # The clean target generated by CMake under cygwin 
+    # The clean target generated by CMake under cygwin
     # always fails for some reason
     #
-	if ! ( uname | grep -q "CYGWIN" ) ; then
-	    "${MAKE_CMD}" -f Makefile clean
-	fi
+  if ! ( uname | grep -q "CYGWIN" ) ; then
+      "${MAKE_CMD}" -f Makefile clean
+  fi
     else
-	eval "${MAKE_CMD} clean > /dev/null 2>&1"
+  eval "${MAKE_CMD} clean > /dev/null 2>&1"
     fi
```

Comment:
The indentation seems wrong, now that your editor has replaced tab with two spaces. It seems that file was using tabs of width 8.

Reply:
The indentation is actually wrong *everywhere* in the two files. Your editor has transformed all <kbd>TAB</kbd> into 2 spaces, instead of 8 spaces.